cou/steps/hypervisor.py

```diff
@@ -236,9 +237,7 @@ def _generate_post_upgrade_steps(
         :rtype: list[PostUpgradeStep]
         """
         steps = []
-        # NOTE(rgildein): Using the reverse order of post-upgrade steps, so these steps starts from
-        #                 subordinate or colocated apps and not nova-compute.
-        for app in self.apps[::-1]:
+        for app in self.apps:
```

Comment:
I still think that running in reverse order make more sense, but if you think that it can be harmful, I'm ok to keep this change.

Reply:
To say the truth I don't understand why we would change the order.
1. We are not using the hypervisor to upgrade subordinates, so the description is outdated
2. Because of the upgrade order, control-plane colocated apps starts to upgrade first and then nova-compute

I think we should keep the way it is and I don't see harm